src/eos_gates_windows.js

```diff
@@ -39,14 +39,17 @@ function loadJSON(path) {
 
 function readWhitelist() {
     let dataDirs = GLib.get_system_data_dirs();
-    dataDirs.forEach(function(dataDir) {
+    return dataDirs.map((dataDir) => {
```

Comment:
Right, that's a good point. In that case, we can just use a regular loop to find the first one.